package.json

```diff
@@ -1,10 +1,9 @@
 {
 	"name": "sugarizer-server",
-	"description": "Sugarizer Server",
+	"description": "Server for Sugarizer application",
 	"version": "1.1.0-alpha",
 	"repository": "https://github.com/llaske/sugarizer-server",
     "author": "Lionel Laské <lionel@sugarizer.org>",
```

Comment:
@NikhilM98 

Could you please solve the indentation issue of this file.

thanks,

Reply:
@tarunsinghal92 I've fixed the indentation issue. Please review the PR